google-cloud-recaptcha_enterprise/samples/create_assessment.rb

```diff
@@ -0,0 +1,70 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START recaptcha_enterprise_create_assessment]
+
+require "google/cloud/recaptcha_enterprise/v1"
+
+def create_assessment
```

Comment:
We should include tests for all samples. That means we'll need to call this method from the test, and pass in actual test data. Generally, the way this is done is to have this method take a bunch of keyword arguments for the test data. You can then document the arguments using Yardoc tags in comments, and then you don't need lines 30-35.